library/Database.re

```
@@ -0,0 +1,7 @@
+open Ezpostgresql;
+
+let conninfo =
+  try (Sys.getenv("DATABASE_URL")) {
+  | Not_found => "postgresql://localhost:5432/summhub_dev"
```

let's throw an exception indeed if DATABASE_URL wasn't defined